src/lib/prompts.ts

```diff
@@ -145,7 +145,11 @@ export default async function prompts({
       name: 'subject',
       placeholder: 'Write a short, imperative tense description of the change.',
       validate(input: string) {
-        if (input.length > commlintRules.subjectMaxLength) {
+        if (input.length === 0) {
+          return `Subject is required.`;
```

Comment:
`subjectEmpty` rule should be read in this condition.
See https://github.com/conventional-changelog/commitlint/blob/master/%40commitlint/rules/src/subject-empty.ts

Reply:
Yes, I ignored this, I will update this code.